tmt/steps/provision/testcloud.py

```diff
@@ -142,6 +132,8 @@ def import_testcloud():
 # VM defaults
 DEFAULT_BOOT_TIMEOUT = 60      # seconds
 DEFAULT_CONNECT_TIMEOUT = 60   # seconds
+TCG_ADDITIONAL_WAIT = 10       # seconds
+TCG_TIMEOUT_COEF = 10          # times
```

Comment:
What does the `TCG_` prefix mean?

Reply:
TCG means the qemu is in SW virtualization mode (kvm-less). Should I change it to NON_KVM?

Reply:
Thanks for clarification, I'd say that the `NON_KVM` prefix would be a bit more self-explanatory. Or adding an explaining comment would also help. Whichever you prefer more.